sdk/src/main/java/io/opentelemetry/sdk/metrics/BaseBoundInstrument.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics;
+
+import io.opentelemetry.metrics.LabelSet;
+
+class BaseBoundInstrument<I extends BaseInstrument<?>> {
```

Comment:
I is not used?

Reply:
that's true. I'll remove it. It feels weird to have the type be absent here, but if we don't actually need it, we don't need it (although, I suspect it'll come back when we plug in aggregators).